pkg/ocm/ocm.go

```diff
@@ -0,0 +1,158 @@
+package ocm
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/openshift/insights-operator/pkg/config"
+	"github.com/openshift/insights-operator/pkg/insights/insightsclient"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/klog/v2"
+)
+
+const (
+	targetNamespaceName = "openshift-config-managed"
+	secretName          = "etc-pki-entitlement" //nolint: gosec
```

Comment:
is this operator owning this secret? what operator manage this?

Reply:
Yes ...I am not sure how the "ownership" is defined, but this secret is created/updated by the Insights Operator.